examples/js/exporters/GLTFExporter.js

```diff
@@ -1028,6 +1004,64 @@ THREE.GLTFExporter.prototype = {
 
 			}
 
+			var forcedIndex = false;
+
+			if ( geometry.index === null && options.forceIndices ) {
+
+				var numFaces = geometry.attributes.position.count;
+				var indices = new Uint32Array( numFaces );
```

Comment:
nit: this should probably be called `numVertices` rather than `numFaces`

Reply:
I prefer

	var indices = new Uint32Array( geometry.attributes.position.count );

	for ( var i = 0, il = indices.length; i < il; i ++ ) {

		indices[ i ] = i;

	}

because we don't need to mind var name hehe.

Reply:
Updated, thanks!